src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -15870,15 +15872,16 @@ namespace OutputReportTabular {
         UnitConv(105).mult = 1.0;
         UnitConv(106).mult = 1.0;
         UnitConv(107).mult = 1.0;
-        UnitConv(108).mult = 0.000238845896627;
-        UnitConv(109).mult = 0.0000004302105;
-        UnitConv(110).mult = 2.2046;
-        UnitConv(111).mult = 1.0;
+        UnitConv(108).mult = 1.0;
+        UnitConv(109).mult = 0.000238845896627;
+        UnitConv(110).mult = 0.0000004302105;
+        UnitConv(111).mult = 2.2046;
         UnitConv(112).mult = 1.0;
         UnitConv(113).mult = 1.0;
-        UnitConv(114).mult = 0.737562149277;
-        UnitConv(115).mult = 1.0;
-        UnitConv(116).mult = 0.09290304;
+        UnitConv(114).mult = 1.0;
+        UnitConv(115).mult = 0.737562149277;
+        UnitConv(116).mult = 1.0;
+        UnitConv(117).mult = 0.09290304;
 
         UnitConv(2).offset = 32.0;
         UnitConv(11).offset = 32.0;
```

Comment:
Noticed unexpected diffs in table convergence limits [here](http://energyplus.s3-website-us-east-1.amazonaws.com/regressions/2019-01/8d7fcb46b41e083a30a872f2b9fc23ca65912c2e-45ac1bb721b1bef6a081ccd753f84c3c6543291a/5ZoneAutoDXVAV/Win64-Windows-7-VisualStudio-14/). The offsets did not get shifted, see line 15889 (lame github won't display the entire file, so a link won't work):
`        UnitConv(80).offset = 7.6736; // 80 is KJ/KG -- should this be multiplied by 1000?
`
Because this didn't get moved to (81), it's being applied to deltaC/deltaF. It looks like all the other members of UnitConv have been shifted appropriately.

Reply:
Good catch!

Not the first time I've fond the UnitConv section to be hard to play with. I'd like to revamp that whole section at some point, use some sort of std::vector<std::pair> to avoid being burned again, make the process of adding a unit easier, and make it easier to read as well.

Reply:
Agreed, this could be improved. A couple of years ago, the units strings for `SetupOutputVariable` were standardized and converted to use [`OutputProcessor::Unit` enum](https://github.com/NREL/EnergyPlus/blob/b0d6191edfecd4a35971336365ef893651e55199/src/EnergyPlus/OutputProcessor.hh#L188-L239). At some point, we would like to do the same to standardize table units and have a common units conversion function that can be used for both tables and output variables (so the eso could be written directly in IP units).